index.html

```diff
@@ -0,0 +1,178 @@
+<!doctype html>
+<html>
+
+<head>
+  <meta charset="UTF-8">
+  <title>Untitled Document</title>
+  <link rel="stylesheet" href="styles/styles.css">
+</head>
+
+<body>
+
+  <div class="container">
+
+    <div id="row-01" class="full-width">
```

Comment:
This is your nav correct? Try putting it within a `nav` tag and putting some containing tag like `div` to represent the seven links in the original page's navbar with appropriate heatmap color.